odoo/addons/base/data/res.country.state.csv

```diff
@@ -1862,3 +1862,40 @@ state_tw_tpc,tw,台北市,TPC
 state_tw_tth,tw,台東縣,TTH
 state_tw_tyc,tw,桃園市,TYC
 state_tw_ylh,tw,雲林縣,YLH
+state_ng_ab,ng,"Abia","NG-AB"
+state_ng_ad,ng,"Adamawa","NG-AD"
+state_ng_ak,ng,"Akwa Ibom","NG-AK"
+state_ng_an,ng,"Anambra","NG-AN"
+state_ng_ba,ng,"Bauchi","NG-BA"
+state_ng_by,ng,"Bayelsa","NG-BY"
+state_ng_be,ng,"Benue","NG-BE"
+state_ng_bo,ng,"Borno","NG-BO"
+state_ng_cr,ng,"Cross River","NG-CR"
+state_ng_de,ng,"Delta","NG-DE"
+state_ng_eb,ng,"Ebonyi","NG-EB"
+state_ng_ed,ng,"Edo","NG-ED"
+state_ng_ek,ng,"Ekiti","NG-EK"
+state_ng_en,ng,"Enugu","NG-EN"
+state_ng_go,ng,"Gombe","NG-GO"
+state_ng_im,ng,"Imo","NG-IM"
+state_ng_ji,ng,"Jigawa","NG-JI"
+state_ng_ka,ng,"Kaduna","NG-KA"
```

Comment:
```suggestion
state_ng_ka,ng,"Kaduna","NG-KD"
```
According to [Wikipedia](https://en.wikipedia.org/wiki/Kaduna_State#:~:text=ISO%203166%20code-,NG%2DKD,-HDI%20(2022))